### ISSUE TITLE
Indentation false error on indented property when parent property has value

### ISSUE THREAD

**Issue by henrahmagix** [sasstools/sass-lint#1008]
This will pass:
```sass
.foo
    margin:
        bottom: 10px
```

But this will fail even though it is correct Sass:
```sass
.foo
    margin: 20px
        bottom: 10px
```

Simply adding a value to a property before nesting it causes an error. sass-lint's indentation rule should allow properties to have value and have nested properties following them.

---

>What version of Sass Lint are you using?

`v1.10.2`

>Please include any relevant parts of your configuration

```yaml
options:
    formatter: stylish
    merge-default-rules: false

rules:

    indentation:
        - 2
        -
            size: 4
```

>What did you do? Please include the actual source code causing the issue.

I nested some properties that also had a value assigned.
```sass
.foo
    margin:
        bottom: 10px
    padding: 10px
        bottom: 20px
    border: 1px solid black
        radius: 0 0 5px 5px
```

>What did you expect to happen?

No errors.

>What actually happened? Please include any error messages given to you by Sass Lint.

Errors on indentation rule for `padding` and `border` (but not `margin`):
```
indentation.sass
  5:9  error  Expected indentation of 4 space but found 8  indentation
  7:9  error  Expected indentation of 4 space but found 8  indentation
```

>If you're using a IDE plugin have you tried the CLI too?

Not using an IDE plugin, but tried the CLI and it erred too, but without the helpful info:
```
/Users/henry/Sites/abbvie-sat-global/web/node_modules/sass-lint/index.js:313
    throw new exceptions.SassLintFailureError(errorCount.count + ' errors were detected in \n- ' + errorCount.files.join('\n- '));
    ^
SassLintFailureError: 2 errors were detected in
- indentation.sass
    at Function.sassLint.failOnError [stack trace...]
```
